packages/zoe/CONTRIBUTING.md

```diff
@@ -46,4 +46,56 @@ Before submitting a pull request, please:
 * `git tag -a zoe-v$VERSION -m "zoe-v$VERSION"`
 * `yarn publish --access public`
 * `git push`
-* `git push --tags`
+* `git push origin zoe-v$VERSION`
+
+Then, once the release has been made, the packages dependent on Zoe
+should be updated in a PR reviewed by the owners of the packages.
+Those packages are:
+* packages/agoric-cli (`packages/agoric-cli/template/contract/package.json`)
+* packages/cosmic-swingset (`packages/cosmic-swingset/package.json`)
+
+To test that that the Zoe update works (in the most basic sense) with cosmic-swingset, do:
+
+1. Run `yarn install` from the workspace root
+2. Run `yarn build` from the workspace root
+3. `cd packages/cosmic-swingset`
+4. `rm -rf t3`
+4. `make scenario3-setup`
+5. `make scenario3-run-client`
+6. Open `http://127.0.0.1:8000/`
+8. `moolaAssayP = E(home.registrar).get("moola_2059");`
```

Comment:
These steps can be made into a script that is run with `agoric deploy`.  That allows you to script any kind of interaction using tildot and the running `ag-solo`.

Reply:
Ah, great. I didn't realize we already had that capability. That's much better. I'll switch to that in another PR.